config.go

```diff
@@ -15,15 +18,46 @@ type Config struct {
 	Slack slack
 }
 
-// ParseConfig reads a ./config.toml file and retruns a parsed config
-func ParseConfig() (*Config, error) {
+// parses a config if it exists, or generates a new one
```

Comment:
apparently the linter complains if you don't start these doc comments with the method name...so it has to read `// LoadConfig ...`